compiler/test/stdlib/random.test.gr

```diff
@@ -0,0 +1,25 @@
+import Random from "random"
+
+let rng1 = Random.make(0xf00L)
+let rng2 = Random.make(0xf00L)
+
+assert Random.nextInt32(rng1) == 833494444l
```

Comment:
Do we like this API instead of `Int32.random()` and `Int32.nextRandom(rng)`?

Reply:
No strong opinion. @ospencer Thoughts?

Reply:
Yeah, I prefer it all existing in the random module.